src/components/HomeButtons.tsx

```diff
@@ -22,75 +23,127 @@ const buttons: THomeButton[] = [
   {
     text: 'Food',
     icon: RestaurantIcon,
-    to: '/food',
+    linkProps: {
+      to: '/food'
+    },
     color: colors.gold
   },
   {
     text: 'Shelter',
     icon: HomeIcon,
-    to: '/shelter',
+    linkProps: {
+      to: '/shelter'
+    },
     color: colors.orangeDark
   },
   {
     text: 'Hygiene',
     icon: HotTubIcon,
-    to: '/hygiene',
+    linkProps: {
+      to: '/hygiene'
+    },
     color: colors.teal
   },
   {
     text: 'Transit',
     icon: BusIcon,
-    to: '/transit',
+    linkProps: {
+      to: '/transit'
+    },
     color: colors.green
   },
   {
     text: 'Resources',
     icon: GroceryStoreIcon,
-    to: '/resources',
+    linkProps: {
+      to: '/resources'
+    },
     color: colors.purple
   },
   {
     text: 'Hotlines',
     icon: CallIcon,
-    to: '/hotlines',
+    linkProps: {
+      to: '/hotlines'
+    },
     color: colors.pink
   },
   {
     text: 'Health',
     icon: HealingIcon,
-    to: '/health',
+    linkProps: {
+      to: '/health'
+    },
     color: colors.red
   },
   {
     text: 'Wifi & Tech',
     icon: WifiIcon,
-    to: '/wifi-and-tech',
+    linkProps: {
+      to: '/wifi-and-tech'
+    },
     color: colors.blue
   },
   {
     text: 'Job Training',
     icon: WorkIcon,
-    to: '/work',
+    linkProps: {
+      to: '/work'
+    },
     color: colors.lavendar
   },
   {
     text: 'Social Services',
     icon: InfoIcon,
-    to: '/social-services',
+    linkProps: {
+      to: '/social-services'
+    },
     color: colors.brown
+  },
+  {
+    text: 'Coordinated Entry',
+    href: 'https://www.bouldercounty.org/homeless/',
+    icon: DoorIcon,
+    color: colors.rosewood,
+    target: '_blank'
   }
 ];
+interface HomeLinkProps extends THomeButton {
+  children: React.ReactElement | React.ReactElement[];
+  key: string;
+}
 
-const HomeLink = styled(Link)`
+const HomeLink = styled((props: HomeLinkProps) => {
```

Comment:
my inclination is to break out the "normal" button and the one-off that opens a link

Reply:
[done](https://github.com/CodeForBoulder/upswyng/pull/87/commits/528bc8c126123fbf3e9a90fd9540612e9e192f19). `<HomeButtons/>` was also getting pretty large so I [separated `<HomeLink/>` and `<HomeButton/>` into different modules](https://github.com/CodeForBoulder/upswyng/pull/87/commits/58c0fb885a79a7c77eaf8e66f990d7bfbf0535c9).